testapp/build.gradle

```diff
@@ -32,22 +32,33 @@ android {
         }
     }
 
-    def appName = "Mini App Sample"
+    def defaultAppName = "Mini App Sample"
+    def debugManifestPlaceholders = [
+            baseUrl                 : property("MINIAPP_SERVER_BASE_URL") ?: "https://www.example.com/",
+            isPreviewMode           : property("IS_PREVIEW_MODE") ?: true,
+            hostAppUserAgentInfo    : property("HOST_APP_UA_INFO") ?: "MiniApp Demo App/$project.version",
+            projectId               : property("HOST_PROJECT_ID") ?: "test-host-project-id",
+            subscriptionKey         : property("HOST_APP_SUBSCRIPTION_KEY") ?: "test-subs-key",
+            adMobAppId              : property("ADMOB_APP_ID") ?: "",
+            appName                 : defaultAppName + " DEBUG"
+    ]
     def stagingManifestPlaceholders = [
             baseUrl                 : property("MINIAPP_SERVER_BASE_URL") ?: "https://www.example.com/",
             isPreviewMode           : property("IS_PREVIEW_MODE") ?: true,
             hostAppUserAgentInfo    : property("HOST_APP_UA_INFO") ?: "MiniApp Demo App/$project.version",
             projectId               : property("HOST_PROJECT_ID") ?: "test-host-project-id",
             subscriptionKey         : property("HOST_APP_SUBSCRIPTION_KEY") ?: "test-subs-key",
-            adMobAppId              : property("ADMOB_APP_ID") ?: ""
+            adMobAppId              : property("ADMOB_APP_ID") ?: "",
+            appName                 : defaultAppName + " STG"
```

Comment:
Looks like `debugManifestPlaceholders` and `stagingManifestPlaceholders` are exactly the same except for the `appName`. So could you just clone it instead of duplicating the properties?

```
def stagingManifestPlaceholders = debugManifestPlaceholders.clone()
stagingManifestPlaceholders.appName = defaultAppName + " STG"
```

Reply:
Thanks, nice idea, I have updated it.